src/berry_mill/kiwrap.py

```diff
@@ -123,7 +123,11 @@ def _get_repokeys(self, reponame: str, repodata: Dict[str, str]) -> str|None:
             return g_path
         else:
             s_url = urljoin(f"{url.scheme}://{url.netloc}/{url.path}/Release.key", "")
-            response = requests.get(s_url, allow_redirects=True)
+            try:
+                response = requests.get(s_url, allow_redirects=True)
+            except Exception as e:
+                log.error(f"Download key error: {e}")
```

Comment:
1. `Error, while downloading the key:`
2. And then? It should probably not continue then, or?

Reply:
The build doesn't need to fail. If all packages are available in the other configured repos, the build works fine, so I would not do a sys.exit.

Reply:
OK, it seems that repos might work sometimes, depends on the setup. Then I would do this way:
1. `Unable to download key:` (server timeout/unavailable etc)
2. Move the logging to the `warning` level